scipy/datasets/download_all.py

```diff
@@ -0,0 +1,47 @@
+#!python3
+"""
+Platform independent script to download all the
+`scipy.dataset` module data files.
+This doesn't require a full scipy build.
+
+Run: python download_all.py <download_dir>
+"""
+
+import argparse
+import pooch
+
+if __package__ is None or __package__ == '':
+    # Running as python script, use absolute import
+    import _registry  # type: ignore
+else:
+    # Running as python module, use relative import
+    from . import _registry
```

Comment:
Note: This is a small hack I used to make it both a python script as well usable in `scipy.datasets` module.
When a python script is run as `main`, the relative `from` imports obviously won't work due to no top-level module. Python complains with `ImportError: attempted relative import with no known parent package <SomePackageBeingImported>`. This should handle that.

I'm open to other possible solutions here.